packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -41,6 +41,8 @@ import {
 } from 'mattermost-redux/utils/post_utils';
 import {getPreferenceKey} from 'mattermost-redux/utils/preference_utils';
 
+import {getEmojiMap} from 'selectors/emojis';
```

Comment:
> Instead of getEmojiMap, you can use the getSystemEmojis and the getCustomEmojis selectors separately there. I can't remember why system emojis are part of the web app, but to allow mattermost-redux to still access those, we had the web app save them as a global variable that mattermost-redux can see. And getSystemEmojis basically returns that variable

Reply:
@hmhealey created a new `makeGetUniqueReactionsToPost` inside the webapp, which uses internally `makeGetReactionsForPost`. Since the latter resides in `mattermost-redux` and doesnt have access to emoji map,  i had to create a new one. Let me know if this is ok and placement of the function is in correct file.